image/pif.ksy

```diff
@@ -0,0 +1,58 @@
+meta:
+  id: pif
+  title: Portable Image Format
+  file-extension: pif
+  license: LGPL-2.1
+  ks-version: 0.10
+  endian: le
+doc-ref: https://github.com/gfcwfzkm/PIF-Image-Format/blob/cc256d5/Specification/PIF%20Format%20Specification.pdf
+seq:
+  - id: header
+    type: header
+  - id: information_header
+    type: information_header
+  - id: color_table
+    size: information_header.len_color_table
+  - id: data
+    size: header.len_file - header._sizeof - information_header._sizeof - information_header.len_color_table
```

Comment:
Is there a reason why you don't use

```suggestion
    size: header.len_image
```

instead? It also seems more correct.

Reply:
Yes, that's actually better. I think that the intention of the author of the format is to not have them diverse, so that should be correct.